src/component/BookList.js

```diff
@@ -0,0 +1,9 @@
+import Book from './Book';
+
+export default function BookList() {
+  return (
+    <ul>
+      <Book />
+    </ul>
+  );
+}
```

Comment:
- [ ] Kindly fetch the books from the store using `useSelector` and add the mapping here, instead of the Book component.